### PR TITLE
Adding support of Zilore API

### PR DESCRIPTION
**Pull request by Habetdin**
Adding support for Zilore.com DNS provider's API

### CODE REVIEW

File: dnsapi/dns_zilore.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+  _debug "Getting TXT records"
+  _zilore_rest GET "domains/${_domain}/records?search_text=$txtvalue&search_record_type=TXT"
+  _debug response "$response"
+
+  if ! _contains "$response" '"ok"' >/dev/null; then
+    _err "Error while getting records list"
+    return 1
+  else
+    _record_id=$(printf "%s\n" "$response" | _egrep_o "\"record_id\":\"[^\"]+\"" | cut -d : -f 2 | tr -d \" | head -n 1)
```
Comment (Neilpang) [inline dnsapi/dns_zilore.sh]: please use our function `_head_n()`  instead of  `head`

**Comment by Neilpang** [comment]
please merge the latest code

File: README.md [hunk 2]
```diff
@@ -315,6 +315,7 @@ You don't have to do anything manually!
 1. zonomi.com DNS API
 1. DreamHost.com API
 1. DirectAdmin API
+1. Zilore API
```
Comment (Neilpang) [inline README.md]: you lost `kinghost`  here.

Comment (Habetdin) [reply README.md]: I don't know how to edit latest versions, not stale ones - resolving this

**Comment by Habetdin** [comment]
Should be up-to-date READMEs now :)

**Pull request merged**